### PR TITLE
[Fix] Fix bug in non-distributed multi-gpu training/testing

### PR DESCRIPTION
**Pull request by AronLin**
Since MMDP does not support non-distributed multi-GPU training, --gpus in train.py lost its role, so it is removed, --gpu-ids is changed to --gpu-id, because only one GPU can be specified for non-distribution training and testing.
If the number of GPUs is more than 1, there will be an assertion error in MMDP because MMDP does not support the situation with more than one GPU.

### CODE REVIEW

**Review by BIGWangYuDong — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

File: tools/train.py [hunk 1]
```diff
@@ -35,15 +35,10 @@ def parse_args():
         help='whether not to evaluate the checkpoint during training')
     group_gpus = parser.add_mutually_exclusive_group()
     group_gpus.add_argument(
-        '--gpus',
+        '--gpu-id',
         type=int,
-        help='number of gpus to use '
```
Comment (ZwwWayne) [inline tools/train.py:40]: We should add deprecation warning of --gpu-ids and --gpus to indicate users that they are deprecated and useless. We should not delete them immediately.

Comment (AronLin) [reply tools/train.py:40]: ok

**Review by ZwwWayne — APPROVED** [review]

**Review by gaotongxiao — COMMENTED** [review]
Never mind... It should have been posted to mmocr :)

**Pull request merged**